build.gradle

```diff
@@ -103,12 +88,11 @@ subprojects { subproject ->
 		testRuntime "org.junit.platform:junit-platform-launcher:$junitPlatformVersion"
 
 		// To support JUnit 4 tests
-		testRuntime "org.junit.vintage:junit-vintage-engine:$junitVintageVersion"
+		testRuntime "org.junit.vintage:junit-vintage-engine:$junitJupiterVersion"
```

Comment:
Is there some guarantee that these versions will always be aligned?

I based this on JUnit's sample.

Reply:
At least that is what I see in their release page: https://github.com/junit-team/junit5/releases

Strange versioning for Milestone though, so, I decided do not upgrade to that one yet.
Although I can reconsider if you insist.